pom.xml

```diff
@@ -4,7 +4,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>nl.mrwouter</groupId>
 	<artifactId>minetopiafarms</artifactId>
-	<version>2.11</version>
+	<version>2.11.2</version>
 	<name>MinetopiaFarms</name>
 	<build>
```

Comment:
Idealiter relocaten we de dependencies om issues te voorkomen, net zoals we eigenlijk alleen de XMaterial class hoeven te shaden. De rest van classes uit XSeries zijn onnodig. Ik zal je een DM sturen met hoe ik het in de MinetopiaSDB plugin heb toegepast.